kratos/utilities/kratos_log.cpp

```diff
@@ -47,11 +47,13 @@ namespace Kratos
         return (*mpInstance);
     }
     
+    // Default constructor
```

Comment:
@pooyan-dadvand are u working on this file?

Reply:
I think that the current in development logger is in a different folder `io`

Reply:
No, this is a deprecated version to be removed. (we are transferring macros from this one)